### PR TITLE
#14829

### PR DESCRIPTION
**Pull request by kaushal07wick**

### CODE REVIEW

**Comment by kaushal07wick** [comment]
I am bit confused on the data generation for meshgrid function.
Any suggestion in the data generation part would be helpful.

**Comment by kaushal07wick** [comment]
@kurshakuz kindly suggest any tips for the data generation part for the Meshgrid function.

**Review by sherry30 — CHANGES_REQUESTED** [review]
Hi,

For generating data, you can take a look at hoe `meshgrid` is being tested in ivy's API. I think it should be very similar.

Thanks

**Comment by kaushal07wick** [comment]
@sherry30 , please look into the test function, i think there is still something missing.

**Comment by kaushal07wick** [comment]
Sure @sherry30 I'll take a look into it.

**Comment by kaushal07wick** [comment]
@sherry30 is there anything missing from the test function, if you can just guide me once., that would be great. Thanks

**Comment by kaushal07wick** [comment]
This was the issue #14829 , it got closed.

**Comment by kaushal07wick** [comment]
meshgrid #14829

**Review by sherry30 — CHANGES_REQUESTED** [review]
Hi, 
Im not sure what you mean by something is missing. Can you point to a specific thing that you think might be missing

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py [hunk 1]
```diff
@@ -1933,6 +1935,39 @@ def test_tensorflow_unique(
...
+    fn_tree,
+    on_device,
+):
+    dtype, arrays = dtype_and_arrays
+    return helpers.test_frontend_function(
+        input_dtypes=dtype,
+        test_flags=test_flags,
+        on_device=on_device,
+        fn_tree=fn_tree,
+        sparse=sparse,
```
Comment (sherry30) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py:1966]: You should send arrays to the test function like `arrays=arrays`

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py [hunk 2]
```diff
@@ -1933,6 +1935,39 @@ def test_tensorflow_unique(
...
+        min_num_dims=1,
+        max_num_dims=1,
+    ),
+    sparse=st.booleans(),
+    indexing=st.sampled_from(["xy", "ij"]),
+    test_with_out=st.just(False),
+)
+def test_meshgrid(
+    *,
+    input_dtypes,
```
Comment (sherry30) [inline ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py:1952]: unsused argument, maybe by mistake?

**Review by sherry30 — CHANGES_REQUESTED** [review]
Some changes requested

File: ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py (hunk 2 above)
Comment (kaushal07wick) [reply ivy_tests/test_ivy/test_frontends/test_tensorflow/test_general_functions.py:1952]: Okay

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-leaves** [comment]
If you are working on an open task, please **edit** the PR description to link to the issue you've created.

For more information, please check [ToDo List Issues Guide](https://unify.ai/docs/ivy/overview/contributing/the_basics.html#todo-list-issues).

Thank you :hugs:

**Pull request closed without merging**